internal/component/loki/source/docker/internal/dockertarget/target.go

```diff
@@ -230,8 +230,6 @@ func (t *Target) StartIfNotRunning() {
 		ctx, cancel := context.WithCancel(context.Background())
 		t.cancel = cancel
 		go t.processLoop(ctx)
-	} else {
-		level.Debug(t.logger).Log("msg", "attempted to start process loop but it's already running", "container", t.containerName)
```

Comment:
Why remove this debug log line? We could just change log message to sth like "starting process loop - already running".

Reply:
with the new retry logic, the line is less relevant and more noisy (currently logs every 5 seconds/target)

Reply:
it's on debug level, so I think the noise wouldn't be too much of a concern, but if it prints every time with this timer than yeah, fine with removing it 👍